amarachukwu-agbo/hello-books
# #158555399 Use appropriate colors for notification alerts

#### What does this PR do?

Change the existing notification alert colors to use the appropriate color in each context

#### Description of Task to be completed?

- Change notification alert to use sweet alert
- Change the color of the alert to use the appropriate color
- Add jsdoc comments

#### How should this be manually tested?

Clone the repo and install application dependencies. Run command  `npm run start:server` and `npm run start:client`. Go to the browser on `127.0.0.1:8080`. Perform actions like signing up and adding a book to favorites. You should see alerts with appropriate colors being displayed.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?

#15855539

#### Screenshots (if appropriate)

N/A

#### Questions:
N/A
